### PR TITLE
Import data

### PR DESCRIPTION
**Pull request by DavidHoltkamp1**
- Got the environment setup correctly. 
- Setup models for all 6 csv files. 
- Create rake task for seeding data 
- Create module to convert unit prices to dollars.

### CODE REVIEW

File: app/models/modules/priceable.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module Priceable
+  def convert_to_dollars
+    self.unit_price = (self.unit_price.to_f / 100)
+  end
+end
```
Comment (DavidHoltkamp1) [inline app/models/modules/priceable.rb]: @mel-rob - hello

Comment (dionew1) [reply app/models/modules/priceable.rb]: My initial thought is that I would change the call-back that you are using to `before_create`. It doesn't look like you are actually using any validations, so maybe that method is never being invoked.

File: lib/tasks/import_csv.rake [hunk 2]
```diff
@@ -0,0 +1,31 @@
+require 'csv'
+
+
+# bundle exec rake import_csv:csv_to_db
+
```
Comment (dionew1) [inline lib/tasks/import_csv.rake]: This looks good. Way to dry up your code here.

File: app/models/modules/priceable.rb (hunk 1 above)
Comment (DavidHoltkamp1) [reply app/models/modules/priceable.rb]: Awesome I will try that, thanks!

**Pull request merged**